frontend/src/setupTests.ts

```diff
@@ -5,6 +5,8 @@ import { Crypto } from '@peculiar/webcrypto'
 import 'locales' // Locales necessary for I18nProvider
 import { server } from './test-utils'
 
+jest.setTimeout(20000)
```

Comment:
Perhaps we can introduce an environment variable with a fallback to a default for this? This way if we want to tune this for different environments, we do not need to make changes in the code.

Reply:
Good idea. I'll do that with a default of 20s.

Reply:
Do test this out on Amplify also with repeated runs.

Reply:
Yup, I'm doing so